pycm/pycm_class_func.py

```diff
@@ -129,17 +129,17 @@ def AGF_calc(TP, FP, FN, TN):
 
 def AGM_calc(TPR, TNR, GM, N, POP):
     """
-    Calculate AGM (Adjusted geometric mean).
+    Calculate Adjusted geometric mean (AGM).
 
     :param TNR: specificity or true negative rate
     :type TNR : float
     :param TPR: sensitivity, recall, hit rate, or true positive rate
     :type TPR : float
     :param GM: geometric mean
     :type GM : float
-    :param N: condition negative
+    :param N:  number of actual negatives
```

Comment:
trailing white-space here, should be:
```
   :param N: number of actual negatives
```
instead of
```
   :param N:  number of actual negatives
```